azure-kit-demo/pom.xml

```
@@ -107,4 +105,4 @@
         </profile>
     </profiles>
 
-</project>
+</project>
```

nit: newline missing
```suggestion
</project>

```

---

Fixed.